web/src/components/App/index.test.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { render, screen } from '@testing-library/react'
 import '@testing-library/jest-dom/extend-expect'
 
-import App from './App'
+import App from '.'
 import { MemoryRouter } from 'react-router-dom'
 
 describe('<App />', () => {
```

Comment:
The tests for `heading` and `list` broke when the DOM changed. I replaced those two checks with [    expect(screen.getAllByRole('link')).toHaveLength(4)](https://github.com/openoakland/woeip/blob/7abc804a8bfbc6821a9c355698aeb552c3a6fc44/web/src/components/App/index.test.tsx#L15)